packages/core/useBase64/index.md

```diff
@@ -0,0 +1,122 @@
+---
+category: Utilities
+---
+
+# useBase64
+
+transform to base64 string
+
+## Usage
+
+```ts
+import { ref, Ref } from 'vue'
+import { useBase64 } from '@vueuse/core'
```

Comment:
Could we simplify down the example in the readme? I think only one example would be enough to showcase the usage